elasticsearch/client/cluster.py

```diff
@@ -102,6 +102,9 @@ def state(self, metric=None, index=None, params=None):
         :arg wait_for_timeout: The maximum time to wait for
             wait_for_metadata_version before timing out
         """
+        if index and not metric:
```

Comment:
I think just for consistency we might want to check against `SKIP_IN_PATH` instead of just the `bool` value. It amounts to the same thing in this case pretty much but would probably be better

Reply:
Yeah agreed.